kt/api-generator/src/main/kotlin/godot/codegen/constants/Classes.kt

```diff
@@ -136,6 +136,20 @@ object GodotTypes {
     )
 }
 
+object GodotMeta {
+    object Int {
+        const val uint32 = "uint32"
+        const val int32 = "int32"
+        const val uint64 = "uint64"
+        const val int64 = "int64"
+    }
+
+    object Float {
```

Comment:
I only meant for Integers to use meta data, not for float.
The reasoning is that the "normal" types in Kotlin are actually Integer and Double. 
In Godot 3, we had no information about the Integer values being 32 or 64 bits which forced us to use Long everywhere, which is against the regular type in Kotlin and was forcing users to put `.toLong()` despite Godot API mostly use 32bits.
Now with meta that issue can finally be solved. But you applied the same treatment to Floating values, turning all Double values to Float. 
The final result is that we reversed the situation for Integers and Floats.

That change was not meant for any gain of performance or memory as the internal buffer and variant are always going to use 64bits values anyway. It was simply to avoid users to cast all the time.